### PR TITLE
DOC-306 New Wordpress Composer Guide

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes # [DOC-306](https://getpantheon.atlassian.net/browse/DOC-306)

## Summary

**[WordPress with Composer on Pantheon](https://pantheon.io/docs/guides/wordpress-composer)** - Created a new guide for Composer-managed WordPress sites on Pantheon.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-7339](https://pr-7339--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `50` | `1` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `98` | `98` | `0` |

Lighthouse Audit Reports:

- [`pr-7339` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/37313/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/37313/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-7339--pantheon-docs.my.pantheonfrontend.website/docs)

**Comment by whitneymeredith** [comment]
[Preview](https://pr-7339--pantheon-docs.my.pantheonfrontend.website/docs/guides/wordpress-composer)

File: source/content/guides/wordpress-composer/02-create-new-basic-wp-composer-site.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+showtoc: true
+permalink: docs/guides/wordpress-composer/create-new-basic-wp-composer-site
+anchorid: create-new-basic-wp-composer-site
+---
+
+This section provides steps for creating a [Composer](/guides/composer)-managed WordPress site on Pantheon. This process creates a new WordPress site with a basic setup. Visit the [Create an Advanced Composer-managed WordPress Site](/guides/wordpress-composer/create-wp-site-composer-ci-auto-test) section of this guide if you want to create a Composer-managed site that uses Continuous Integration and automated testing.
+
+ You can review the Pantheon-maintained [WordPress Composer Repo](https://github.com/pantheon-systems/wordpress-composer/) for more details on the basic Composer-managed WordPress site creation process.
+
+1. [Spin up a new WordPress site](/create-sites) on Pantheon.
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/02-create-new-basic-wp-composer-site.md:18]: Creating a site through dashboard won't give you a composer managed version of Wordpress. It's a basic wordpress installation based on https://github.com/pantheon-systems/wordpress/tree/master

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+    - You must use the [Build Tools 3.x release](https://github.com/pantheon-systems/terminus-build-tools-plugin/tree/3.x) if you are using **Terminus 3**.
+
+    - You must use the [Build Tools 2.x release](https://github.com/pantheon-systems/terminus-build-tools-plugin/tree/2.x) if you are using **Terminus 2**.
+
+- [PHP version](/php-versions#verify-current-php-versions) 7.2 or greater
+
+## Create Your Site
+
+1. Use the [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin) to copy the Pantheon-maintained [WordPress Composer repository](https://github.com/pantheon-systems/example-wordpress-composer).
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:28]: There's no "how" in this guide for this alternative.

`terminus build:project:create wp` should be used as a very basic, but several options are available

Comment (kporras07) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:28]: There's no "how" in this guide for this alternative.

`terminus build:project:create wp` should be used as a very basic, but several options are available

Comment (kporras07) [reply source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:28]: Note that using build tools will spin up the site based on https://github.com/pantheon-systems/example-wordpress-composer

Comment (kporras07) [reply source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:28]: Note that using build tools will spin up the site based on https://github.com/pantheon-systems/example-wordpress-composer

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+        - `tests/behat/tests/behat/features/visit-homepage.feature` is a Behat test file that visits the homepage and verifies a `200` response.
+
+        - `tests/behat/tests/behat/features/admin-login.feature` is a Behat test file that logs into the WordPress dashboard as an administrator and verifies access to new user creation.
+
+        - `tests/behat/tests/behat/features/admin-login.feature` is a Behat test file that logs into the WordPress dashboard as an administrator, updates the `blogname` and `blogdescription` settings, clears the Pantheon cache, visits the home page, and verifies how the updated blog name and description appear.
+
+
+## GitHub Actions
+
+This section provides information on how to enable GitHub Actions for an existing project.
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:152]: If Github Actions was passed as the selected CI when creating the build tools site then it will be configured and this won't be necessary.

If you're working with an existing (non build tools) site and want to add Github Actions then it's really not super easy to do

If you're working on an existing build tools site in another CI (e.g. CircleCI) and want to move to Github Actions, this _may_ work but this needs some testing

Comment (kporras07) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:152]: If Github Actions was passed as the selected CI when creating the build tools site then it will be configured and this won't be necessary.

If you're working with an existing (non build tools) site and want to add Github Actions then it's really not super easy to do

If you're working on an existing build tools site in another CI (e.g. CircleCI) and want to move to Github Actions, this _may_ work but this needs some testing

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+categories: [develop]
+tags: [wordpress]
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code.
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:14]: "Bedrock is a central feature of Composer" not sure what does this mean. Maybe of "Composer in the Wordpress ecosystem"?

Comment (kporras07) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:14]: "Bedrock is a central feature of Composer" not sure what does this mean. Maybe of "Composer in the Wordpress ecosystem"?

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
+
+## Create a New Pantheon WordPress Site with Bedrock
+
+1. Run the following command in Terminus:
+
+    ```bash
+    composer create-project your_site
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:21]: This command is probably incomplete

Comment (kporras07) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:21]: This command is probably incomplete

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+---
+title: Using Bedrock to Create Your WordPress Pantheon Site
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:2]: In general, I don't think we should be proposing people to use Bedrock upstream yet as it's not production-ready and development is still pending/in-progress.

Comment (kporras07) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:2]: In general, I don't think we should be proposing people to use Bedrock upstream yet as it's not production-ready and development is still pending/in-progress.

File: source/content/guides/wordpress-composer/06-wordpress-ic.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+
+## Create Your WordPress Integrated Composer Site
+
+Follow the steps in this section to create a new WordPress site using Integrated Composer and Upstream.
+
+1. Fork the [Pantheon-maintained repository](https://github.com/pantheon-upstreams/wordpress-project).
+
+1. [Connect your repository](/create-custom-upstream#connect-repository-to-pantheon) to Pantheon.
+
+1. Edit the **vendor** name used in the `upstream-config` directory.
```
Comment (kporras07) [inline source/content/guides/wordpress-composer/06-wordpress-ic.md:36]: Is this really needed? I think there is no harm in leaving it as "pantheon-upstreams" and it will make easier to adopt possible updates in their custom upstreams in the future

Comment (kporras07) [inline source/content/guides/wordpress-composer/06-wordpress-ic.md:36]: Is this really needed? I think there is no harm in leaving it as "pantheon-upstreams" and it will make easier to adopt possible updates in their custom upstreams in the future

File: source/content/guides/wordpress-composer/04-bedrock-composer.md (hunk 4 above)
Comment (jazzsequence) [reply source/content/guides/wordpress-composer/04-bedrock-composer.md:14]: Agree with @kporras07. _Bedrock_ isn't a central feature of Composer as Composer exists far and wide outside of the WordPress-specific implementation in Bedrock. I would also argue that "Composer does not fully support WordPress" is not entirely accurate. I would rephrase this to:

```suggestion
WordPress does not natively support [Composer](https://getcomposer.org/), however, Bedrock is a WordPress-specific framework for using Composer on WordPress sites. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
```

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+    - Your `.env` will be created. Your new `.env` file is smaller than the original Bedrock `.env` file because Pantheon automatically supplies many of the environment variables that are usually stored in the `.env` file.
+
+1. Replace all placeholder keys with the [new keys](https://roots.io/salts.html).
+
+1. Use [SFTP or git](/guides/wordpress-git/) to push changes directly to your Pantheon site.
+
+## See Also
+
+- [Pantheon WordPress with Bedrock Recommended GitHub Repository](https://github.com/pantheon-systems/wordpress-bedrock-recommended)
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:71]: This is an unsupported prototype repository that probably we shouldn't link out to until it's in a more finalized state.

File: source/content/guides/wordpress-composer/04-bedrock-composer.md (hunk 5 above)
Comment (jazzsequence) [reply source/content/guides/wordpress-composer/04-bedrock-composer.md:21]: The Bedrock docs list `composer create-project roots/bedrock` as the start step for installing Bedrock. This command pulls in `roots/bedrock` to be the framework for the Composer-ized project, so replacing `roots/bedrock` with `your_site` would break/not pull in Bedrock.

Comment (jazzsequence) [reply source/content/guides/wordpress-composer/04-bedrock-composer.md:21]:
```suggestion
    composer create-project roots/bedrock
```

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/bedrock-composer
+anchorid: bedrock-composer
+---
+
+[Composer](https://getcomposer.org/) does not fully support WordPress, however, Bedrock is a central feature of Composer. [Bedrock](https://roots.io/bedrock/) is a custom WordPress project structure that helps simplify working on large projects with multiple plugins and custom code. 
+
+## Create a New Pantheon WordPress Site with Bedrock
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:16]: It feels like a step is missing here. How do users get the site to start out? When using `composer create-project` it is expected that this is the first step in setting up a site in an empty repository. This isn't possible if we've created a site first as described in the "Create a Basic Composer-managed WordPress Site" doc. Even if we deleted everything from that site after creating and cloning, we'd still have the `.git` directory left (or we'd break the ability to commit and push).

There is an "Empty WordPress" upstream that was created for this purpose, is the intention that they start out with that? (I suspect that would still run into issues if cloned because the directory would still not be empty when running `composer create-project`.)

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+1. Store your `.env` outside of the project root on the Pantheon server. 
+
+    - Bedrock is configured to look for the `.env` file in the `files/private` directory by default. This is also the location where Pantheon stores all files that shouldn't be accessible to the public.
+
+1. Commit the `vendor` and `web` directories.
+
+    - Pantheon does not support using Composer to install dependencies that are deployed to Pantheon servers. You must commit the entire `vendor` and `web` directories into your Git repo to ensure that your dependencies are installed for the new project.
+
+1. Convert the `web/app/uploads` directory to a [symlink](https://github.com/git-for-windows/git/wiki/Symbolic-Links) that points to the `files` directory.
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:34]: The link is a specific Windows-only method of creating symbolic links that will not work on Mac/*nix machines. Perhaps link to documentation for both?

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+
+1. Run the following command in Terminus:
+
+    ```bash
+    composer create-project your_site
+    ```
+
+1. Update the [environment variables](https://docs.roots.io/bedrock/master/environment-variables/#wp-env) in your project's `.env` file.
+
+1. Store your `.env` outside of the project root on the Pantheon server.
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:26]: this step seems out of order. All the other steps relating to `.env` are below the step to commit `vendor` and `web`. Also, for the record, I didn't need to _create_ a `.env` file, I just had one, presumably as part of the Bedrock initialization step.

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+1. Commit the `vendor` and `web` directories.
+
+    - Pantheon does not support using Composer to install dependencies that are deployed to Pantheon servers. You must commit the entire `vendor` and `web` directories into your Git repo to ensure that your dependencies are installed for the new project.
+
+1. Convert the `web/app/uploads` directory to a [symlink](https://github.com/git-for-windows/git/wiki/Symbolic-Links) that points to the `files` directory. 
+
+    - This is necessary because Pantheon stores media files in the `files` directory instead of the `uploads` directory.
+
+1. Create the initial `.env` file.
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:38]: As mentioned above, this wasn't necessary for me in testing, I just needed to move it to `files/private`

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+    - This is necessary because Bedrock uses environment variables to manage sensitive credentials. These environment variables usually come from multiple sources. The most important variable is the `.env` file which Bedrock uses instead of the standard WordPress `wp-config.php` file. Pantheon was not designed to create the initial `.env` file that your Bedrock site requires. You must create it yourself and upload it to the Pantheon server. The easiest way to do that is by connecting to your Pantheon server using FTP and creating the `.env` file. 
+
+1. Upload the newly created `.env` file by connecting to the Pantheon server using an FTP.
+
+    - To find your connection information: Navigate to the site's **Dev** environment in Pantheon > click **Connection Info** > scroll to the bottom of the menu to view your SFTP connection information.
+
+1. Connect to your Pantheon site using the SFTP credentials located in the previous step > navigate to the `files` directory > create a `private` directory > create the `.env` file in the `private` directory by running the following command:
+
+    ```bash
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:48]: It might be more helpful to provide an example of what the `.env` file looks like since this looks like Bedrock default. None of the credentials are exposed because we use global server variables.

```bash
DB_NAME=$_ENV['DB_NAME']
DB_USER=$_ENV['DB_USER']
DB_PASSWORD=$_ENV['DB_PASSWORD']

# Optionally, you can use a data source name (DSN)
# When using a DSN, you can remove the DB_NAME, DB_USER, DB_PASSWORD, and DB_HOST variables
# DATABASE_URL='mysql://database_user:database_password@database_host:database_port/database_name'

# Optional database variables
DB_HOST=$_ENV['DB_HOST'].':'.$_ENV['DB_PORT']
# DB_PREFIX='wp_'

WP_ENV='development'
WP_HOME='http://example.com'
WP_SITEURL="${WP_HOME}/wp"

# Specify optional debug.log path
# WP_DEBUG_LOG='/path/to/debug.log'

# Generate your keys here: https://roots.io/salts.html
AUTH_KEY=$_ENV['AUTH_KEY']
SECURE_AUTH_KEY=$_ENV['SECURE_AUTH_KEY']
LOGGED_IN_KEY=$_ENV['LOGGED_IN_KEY']
NONCE_KEY=$_ENV['NONCE_KEY']
AUTH_SALT=$_ENV['AUTH_SALT']
SECURE_AUTH_SALT=$_ENV['SECURE_AUTH_SALT']
LOGGED_IN_SALT=$_ENV['LOGGED_IN_SALT']
NONCE_SALT='generateme'
```

File: source/content/guides/wordpress-composer/04-bedrock-composer.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+    LOGGED_IN_SALT='generateme'
+    NONCE_SALT='generateme'
+    ```
+
+    - Your `.env` will be created. Your new `.env` file is smaller than the original Bedrock `.env` file because Pantheon automatically supplies many of the environment variables that are usually stored in the `.env` file.
+
+1. Replace all placeholder keys with the [new keys](https://roots.io/salts.html).
+
+1. Use [SFTP or git](/guides/wordpress-git/) to push changes directly to your Pantheon site.
+
```
Comment (jazzsequence) [inline source/content/guides/wordpress-composer/04-bedrock-composer.md:68]: There are more steps missing here as well. After following the steps, I had a site created but I was not able to access the admin, see https://dev-cr-bedrock-docs-test.pantheonsite.io/

I am not sure exactly what's missing without further testing, but this workflow, even with the adjustments I made to try to get it working, does not end with a working WordPress site. 😞

File: source/content/guides/wordpress-composer/01-introduction.md [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Review [Managing Core as Project Dependency](/guides/composer#managing-core-as-a-project-dependency).
+
+1. Review [Custom Upstream Workflow](/guides/composer#custom-upstream-workflow).
+
+## See Also
+
+- [Composer Fundamentals and WebOps Workflows](/guides/composer)
+
+- [Convert a Standard Drupal 8 Site to a Composer-Managed Site](/guides/composer-convert)
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/01-introduction.md:52]: Do we want to cross-reference from WordPress to Drupal pages here?

File: source/content/guides/wordpress-composer/02-create-new-basic-wp-composer-site.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+contributors: [whitneymeredith]
+layout: guide
+showtoc: true
+permalink: docs/guides/wordpress-composer/create-new-basic-wp-composer-site
+anchorid: create-new-basic-wp-composer-site
+---
+
+This section provides steps for creating a [Composer](/guides/composer)-managed WordPress site on Pantheon. This process creates a new WordPress site with a basic setup. Visit the [Create an Advanced Composer-managed WordPress Site](/guides/wordpress-composer/create-wp-site-composer-ci-auto-test) section of this guide if you want to create a Composer-managed site that uses Continuous Integration and automated testing.
+
+ You can review the Pantheon-maintained [WordPress Composer Repo](https://github.com/pantheon-systems/wordpress-composer/) for more details on the basic Composer-managed WordPress site creation process.
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/02-create-new-basic-wp-composer-site.md:16]: I think that most users would be confused if they examined the wordpress-composer repository. In any event, I think that this page should be removed until we have a supported (Bedrock-based) Composer-managed WordPress upstream.

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 17]
```diff
@@ -0,0 +1,215 @@
+---
+title: Create an Advanced Composer-managed WordPress Site
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:2]: Maybe the title should contain "Continuous Integration"?

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 18]
```diff
@@ -0,0 +1,215 @@
...
+
+    - This site will be based on the Pantheon-maintained [WordPress Composer repository](https://github.com/pantheon-systems/example-wordpress-composer).
+
+1. Review the sections below for important information about your site, including an explanation of the directory structure, Continuous Integration functions, and essential configuration actions.
+
+## Review Important Directories and Update File Paths
+
+### /web Directory
+
+Your site is stored and served from the `/web` subdirectory in the `pantheon.yml` file. You must store your website in this subdirectory for a Composer-based workflow. Placing your website in the subdirectory also allows you  to store tests, scripts, and other files related to your project in your repo without affecting your web document root. It also provides additional security by preventing web access through Pantheon. Your files may still be accessible from your version control project if it is public. See the [`pantheon.yml` documentation](/pantheon-yml#nested-docroot) for details.
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:51]: ...from the `/web` subdirectory *next to* the `pantheon.yml` file.

It also provides additional security by preventing web access *to files outside of the document root* through Pantheon.

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 19]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Verify that your website is stored in the `/web` subdirectory.
+
+### /web/wp Directory
+
+Your directories and files within the `/web` directory are stored in different locations compared to a default WordPress installation. [WordPress allows installing WordPress core in its own directory](https://wordpress.org/support/article/giving-wordpress-its-own-directory/), which is necessary when installing WordPress with Composer. The overall layout of directories in the repo is similar to [Bedrock](https://github.com/roots/bedrock).
+
+1. Move the `WP_SITEURL` to the `/web/wp` directory to allow WordPress core functions to work correctly.
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:59]: Isn't this already set up for the user if they create their site via `build:project:create`?

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 20]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Move the `WP_SITEURL` to the `/web/wp` directory to allow WordPress core functions to work correctly. 
+
+1. Review the `/web/wp-config.php` file for key settings and move other files to the to the `/web/wp` directory as necessary.
+
+### composer.json File
+
+This project uses Composer to manage third-party PHP dependencies. Some files, such as `web/wp` may not be visible in the repository. This is because WordPress core and its plugins are installed via Composer and ignored in the `.gitignore` file.
+
+A custom, [Composer version of WordPress for Pantheon](https://github.com/pantheon-systems/wordpress-composer/) is used as the source for WordPress core.
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:67]: This isn't really a custom version of WordPress; this is a copy of the standard version of WordPress with a composer.json file added. It is maintained automatically via automation.

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 21]
```diff
@@ -0,0 +1,215 @@
...
+
+Non-WordPress dependencies are downloaded to the `/vendor` directory.
+
+1. Place all dependencies in the **require** section of your `composer.json` file. 
+
+    - This includes dependencies that are only used in non-Live environments. All dependencies in the **require** section are pushed to Pantheon.
+
+1. Place all dependencies that are not a part of the web application but are necessary to build or test the project in the **require-dev** section.
+
+    - Some example dependencies are `php_codesniffer` and `phpunit`. Dev dependencies are not deployed to Pantheon.
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:79]: Dev dependencies are deployed to dev and multidev environments, but are not deployed to the test and live environments.

File: source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md [hunk 22]
```diff
@@ -0,0 +1,215 @@
...
+
+1. Add the following secrets to the Github Actions configuration:
+
+    `ADMIN_EMAIL`
+    `ADMIN_PASSWORD`
+    `ADMIN_USERNAME`
+    `TERMINUS_TOKEN`
+    `TERMINUS_SITE`
+    `SSH_PRIVATE_KEY`
+    `GH_TOKEN`
```
Comment (greg-1-anderson) [inline source/content/guides/wordpress-composer/03-create-wp-site-composer-ci-auto-test.md:176]: We should see if we can get build:project:repair to switch CI providers. Until that's confirmed, the advice above looks fine.

File: source/content/guides/wordpress-composer/04-bedrock-composer.md (hunk 6 above)
Comment (greg-1-anderson) [reply source/content/guides/wordpress-composer/04-bedrock-composer.md:2]: Agree, please remove this section until we have a supported Bedrock upstream.

**Comment by wordsmither** [comment]
lgtm
